sql/src/test/java/io/crate/analyze/BaseAnalyzerTest.java

```diff
@@ -322,11 +327,24 @@ protected AnalyzedStatement analyze(String statement, Object[][] bulkArgs) {
 
     @Before
     public void prepareModules() throws Exception {
+        threadPool = new ThreadPool("testing");
```

Comment:
maybe mocking a ThreadPool and returning a directExecutor or a singleThreadExecutor on executor(String) would be less expensive, wouldn't it?